test/dct_tests/src/test_dctXX_inverse.c

```diff
@@ -27,20 +27,13 @@ TEST_SETUP(dctXX_inverse) { fflush(stdout); }
 TEST_TEAR_DOWN(dctXX_inverse) {}
 
 
-
 #define MAX_PROC_FRAME_LENGTH_LOG2 9
 #define MAX_PROC_FRAME_LENGTH (1<<MAX_PROC_FRAME_LENGTH_LOG2)
 
 #define MIN_DCT_N_LOG2  (3)
 #define LOOPS_LOG2  (0)
 
 
-
```

Comment:
why is SMOKE_TEST not used here? Is it because `LOOPS_LOG2`  is already 0?

Reply:
Yes, we can't go any lower